src/lib/UniswapV2/contracts.ts

```diff
@@ -0,0 +1,32 @@
+import { ethers } from "ethers";
+import { CHAIN_ID, LISTEN_FOR_TRANSACTIONS } from "../../enums";
+import { IProtocolContractDefinitions } from "../../types";
+import UniswapV2RouterABI from "./abis/UniswapV2Router.json";
+
+export enum CONTRACT_ENUM {
+  ROUTER = "UniswapV2Router02",
+}
+
+export enum EVENT_ENUM {
+  SWAP = "Swap",
+}
+
+export const contracts: IProtocolContractDefinitions = {
+  [CONTRACT_ENUM.ROUTER]: {
+    interface: new ethers.Interface(UniswapV2RouterABI),
+    deployments: {
+      [CHAIN_ID.ARBITRUM]: {
+        address: "0x4752ba5dbc23f44d87826276bf6fd6b1c372ad24",
+        listenForTransactions: [LISTEN_FOR_TRANSACTIONS.INCOMING],
+      },
+    },
+    events: {
+      [EVENT_ENUM.SWAP]: {
+        signature: ethers.id("Swap(address,uint256,uint256,uint256,uint256,address)"),
```

Comment:
Can you hardcode this value? Look up this log in the explorer and use the topics[0]
Reason is having the hardcoded signature makes it easier for us to search the logs in our db if we need to

If you want to store the signature you can like you have currently just add a new field